clients/python/setup.py

```diff
@@ -0,0 +1,33 @@
+import setuptools
+
+requirements = [
+    "aiohttp>=3.7.1,<3.8.0",
+    "gql>=3.0.0a5",
+    "python-slugify>=4.0.1",
+]
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="tracker_client",
+    version="1.0.0-a",
+    author="Thomas Nickerson",
+    author_email="contact@cyber.gc.ca",
+    description="A Python client for the GoC Tracker API",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/canada-ca/tracker",
```

Comment:
Because we have a mono repo it may be better to directly link to the clients directory: `https://github.com/canada-ca/tracker/tree/master/clients/python`

Reply:
Makes sense, I’ll make the change. Thanks!